wrappers/windres-wrapper.c

```diff
@@ -322,7 +334,7 @@ int _tmain(int argc, TCHAR* argv[]) {
         }
 
         arg = 0;
-        exec_argv[arg++] = concat(dir, _T("llvm-rc"));
+        exec_argv[arg++] = concat(dir, _T(LLVM_RC));
```

Comment:
I don't think these concatenations would work out properly if you hardcode a prefix inbetween.

And I presume that the corresponding prefix manipulations for clang (both for deducing an architecture from the windres wrapper's name, and for appending the clang binary's name to the directory) also will need adjustments for your usecase (although possibly as a separate patch).